### PR TITLE
feat(autocomplete-select): remove hint icon

### PR DESCRIPTION
**Pull request by rodrigosakamoto**
## Description
[[Driver List] Name/ID field is autocomplete instead of a dropdown](https://trello.com/c/SyPy1bQF/1099-driver-list-name-id-field-is-autocomplete-instead-of-a-dropdown)
add prop to autocomplete select to remove hint icon

### Desktop
Screenshot from 2022-09-13 11-29-49
## Types of changes

- [ ] Fix (non-breaking change which fixes an issue)
- [ ] Feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Build (Changes that affect the build system or dependencies)
- [ ] Chore (Other changes that do not modify src or test files)
- [ ] CI (Changes to our CI configration and scripts)
- [ ] Docs (Changes to documentation files)
- [ ] Style (Changes to coding style, like linters and prettier)
- [ ] Refactor (Changes that do not add new functionality or alter current ones)
- [ ] Performance (Performance improvements)
- [ ] Test (Adding or changing tests or the testing setup)
- [ ] Revert (Reverting a previous change)

## Checklist:

- [ ] I have made sure my changes work properly on iOS devices running Safari.
- [ ] I have made sure my changes work properly on Android devices running Chrome.
- [ ] I have made sure my changes work properly on Desktop.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

**Comment by logrock-eng** [comment]
# Netlify Preview Deployed 

 Preview: https://105--logrock-pebbles.netlify.app 

 ## View on Browserstack 

 https://live.browserstack.com/dashboard#os=iOS&os_version=15.0&device_browser=safari&zoom_to_fit=true&full_screen=true&url=https://105--logrock-pebbles.netlify.app&speed=1

**Comment by logrock-eng** [comment]
# Netlify Preview Deployed 

 Preview: https://105--logrock-pebbles.netlify.app 

 ## View on Browserstack 

 https://live.browserstack.com/dashboard#os=iOS&os_version=15.0&device_browser=safari&zoom_to_fit=true&full_screen=true&url=https://105--logrock-pebbles.netlify.app&speed=1

File: lib/components/Select/Select.tsx [hunk 1]
```diff
@@ -16,6 +16,7 @@ const Select = <ItemType extends BaseItemType>({
   maxHeight,
   multiSelect,
   smartExpand = false,
+  hideHintIcon,
```
Comment (cld-santos) [inline lib/components/Select/Select.tsx]: you can set as false by default

**Comment by logrock-eng** [comment]
# Netlify Preview Deployed 

 Preview: https://105--logrock-pebbles.netlify.app 

 ## View on Browserstack 

 https://live.browserstack.com/dashboard#os=iOS&os_version=15.0&device_browser=safari&zoom_to_fit=true&full_screen=true&url=https://105--logrock-pebbles.netlify.app&speed=1

**Review by cld-santos — APPROVED** [review]

**Comment by logrock-eng** [comment]
# Netlify Preview Deployed 

 Preview: https://105--logrock-pebbles.netlify.app 

 ## View on Browserstack 

 https://live.browserstack.com/dashboard#os=iOS&os_version=15.0&device_browser=safari&zoom_to_fit=true&full_screen=true&url=https://105--logrock-pebbles.netlify.app&speed=1

**Comment by logrock-eng** [comment]
# Netlify Preview Deployed 

 Preview: https://105--logrock-pebbles.netlify.app 

 ## View on Browserstack 

 https://live.browserstack.com/dashboard#os=iOS&os_version=15.0&device_browser=safari&zoom_to_fit=true&full_screen=true&url=https://105--logrock-pebbles.netlify.app&speed=1

**Pull request merged**